plugins/org.eclipse.papyrus.uml.diagram.sequence.runtime/src/org/eclipse/papyrus/uml/diagram/sequence/runtime/internal/edit/parts/MessageEditPart.java

```diff
@@ -224,6 +243,43 @@ protected void refreshRouterChange() {
 		// No bendpoints in messages to refresh
 	}
 
+	@Override
+	protected void handlePropertyChangeEvent(PropertyChangeEvent event) {
+
```

Comment:
This one is not able to differentiate if this property is changed only for feedback or for the "real" connection. So it is always computing the router based on the routing style in the model. We do not change this style until the element is finally connected, to avoid affecting the model and issues during cancel. 
So I prefer to avoid this call, and let the feedback management to update the router for the feedback itself.